examples/uidl-samples/cms-project.json

```diff
@@ -990,11 +1015,132 @@
             "type": "conditional",
             "content": {
               "node": {
-                "type": "element",
+                "type": "cms-item",
                 "content": {
-                  "elementType": "cms-collection-item",
-                  "referencedStyles": {},
-                  "children": [],
+                  "node": {
+                    "type": "element",
+                    "content": {
+                      "elementType": "cms-item",
+                      "semanticType": "cms-item",
+                      "ctxId": "396bec26-fced-4004-9dfd-a7e82417d9c4",
+                      "attrs": {
+                        "value": {
+                          "type": "dynamic",
+                          "content": {
+                            "referenceType": "prop",
+                            "id": "BooksEntity.attributes"
+                          }
+                        }
+                      },
+                      "children": [
+                        {
+                          "type": "element",
+                          "content": {
+                            "elementType": "fragment",
```

Comment:
Because of this I think the exported code looks like this:
` <BooksContext.Provider value={props.BooksEntity.attributes}>
        <fragment>
          <BooksDetailsContent></BooksDetailsContent>
        </fragment>
      </BooksContext.Provider>`

Reply:
Discussed: will be handled in another PR